### PR TITLE
Created a confirmation dialog, made changes and organized the code

### PR DESCRIPTION
**Pull request by MigDinny**
This solves: #808 

The code was a mess and I idented it and now it must be fine.

Created a confirm message when clicking on submit to avoid accidental clicks. The message is now red.

The successful message is now an Alert() .

There is an issue on line 57. There must be ONLY one bracket at the beginning ( } ) and the code needs 2 of them to work. If i remove one of them, the tab is going to be Undefined and it just doesn't load. I really don't know why this is happening.

Now:

**Normal state**
image

**Confirmation dialog**
image

There is another 2 dialogs when clicking Ok but this must be enough.

### CODE REVIEW

**Comment by teryhill** [comment]
@aethelwulffe You will want to look at this

**Comment by teryhill** [comment]
@aethelwulffe  bump

**Comment by robbyoconnor** [comment]
Going to approve their [task](https://codein.withgoogle.com/dashboard/task-instances/4863693995114496/), @MigDinny please fix any issues that may pop up. 

Also going forward, we'd like to see you work on the multiple bugs per task.

File: modules/MIPS/truncate_reports.php [hunk 1]
```diff
@@ -1,43 +1,63 @@
...
+	 * 
+	 * IMPORTANT:
+	 * This whole feature needs to be moved to a integrated admin form, and naturally use css buttons and translations.
+	 */
+	require_once '../../interface/globals.php';
+	include_once("$srcdir/api.inc");
+	include_once("$srcdir/acl.inc");
+
+	// Warning message created here and echoed on its place. The single ' is shown as ` don't know why. This must be fixed.
+	$reportDeleteWarning = xlt("This will delete all old reports and it's irreversible!");
```
Comment (aethelwulffe) [inline modules/MIPS/truncate_reports.php]: "This will irreversibly delete all existing reports!"

File: modules/MIPS/truncate_reports.php [hunk 2]
```diff
@@ -1,43 +1,63 @@
...
-</form>
+				// I JUST REALIZED THAT IF I REMOVE ONE OF THE TWO BRACKETS BELOW (}}) THE CODE DOESNT WORK!
+				// It must be some kind of PHP error or cache, i don't really know. 
+			}} else {
+				echo "<script>alert('".$accessDeniedMessage."');</script>";
+			}
+				
+		?>
+	</form>
+</html>
```
Comment (aethelwulffe) [inline modules/MIPS/truncate_reports.php]: Newline...

File: modules/MIPS/truncate_reports.php (hunk 1 above)
Comment (MigDinny) [reply modules/MIPS/truncate_reports.php]: done

**Comment by teryhill** [comment]
@MigDinny clean up the pr please

**Comment by MigDinny** [comment]
@teryhill what you mean?

**Comment by aethelwulffe** [comment]
You "updated" your branch, which means you pulled in all the intervening commits.
Basically, you might want to rewind and play your changes on top...
Look up.  See the list of commits in this PR....lot of stuff.
So, you could do a new branch, playing your changes on top...which is often the simple answer.  You could also figure out how to actually rebase this properly...which is the very very hard answer.
  Having merge commits from an update is considered bad here...don't ask me why.  It seems like a supported function...but they don't like it here, so we clean up and only have new work in the branch...without updating the branch in the middle of working on it.

**Comment by MigDinny** [comment]
@aethelwulffe https://github.com/LibreHealthIO/LibreEHR/pull/835 what about this?

**Comment by aethelwulffe** [comment]
closing this, going to new PR.

**Pull request closed without merging**